meson.build

```diff
@@ -14,4 +14,15 @@ src = [
   'src/charstream.c',
   ]
 
+
 executable('ubsh', src, include_directories: inc)
+
+unit_tests_src = [
+  'src/charstream.c',
+  'tests/charstream.c'
+  ]
```

Comment:
Maybe we could come up with something better to add every source file but main and the test source files? We can concatenate arrays with meson, that might be a way

Reply:
Good idea, we should look into it

Reply:
Fixed